x/shield/keeper/pool.go

```diff
@@ -41,122 +123,70 @@ func (k Keeper) CreatePool(ctx sdk.Context, creator sdk.AccAddress,
 		return types.Pool{}, types.ErrSponsorAlreadyExists
 	}
 
-	if !k.ValidatePoolDuration(ctx, poolLifeTime) {
+	// FIXME: This is incorrect. Should make sure the protection period is 21x days.
+	if !k.ValidatePoolDuration(ctx, protectionPeriod) {
```

Comment:
Each time create / update pool the protection is 21 days. There is no multiple of 21 days for backend. It only exists in social negotiation.